CodeGen/tests/test_codegen_on_xeon.sh

```diff
@@ -41,6 +42,7 @@ function start_services() {
         echo "using image repository $IMAGE_REPO and image tag $IMAGE_TAG"
         sed -i "s#image: opea/codegen:latest#image: opea/codegen:${IMAGE_TAG}#g" docker_compose.yaml
         sed -i "s#image: opea/codegen-ui:latest#image: opea/codegen-ui:${IMAGE_TAG}#g" docker_compose.yaml
+        sed -i "s#image: opea/codegen-react-ui:latest#image: opea/codegen-react-ui:${IMAGE_TAG}#g" docker_compose.yaml
```

Comment:
Please add the same line for `CodeGen/tests/test_codegen_on_gaudi.sh`

Reply:
added `CodeGen/tests/test_codegen_on_gaudi.sh` file